### ISSUE TITLE
Dummy support for 0.6.5

### ISSUE THREAD

**Issue by ChillerDragon** [ddnet/ddnet#1698]
Dummys get really glitchy on latest vanilla. Probably they can't handle the anti spoof. Graphic bugs connection problems and falling through walls -.-

**Comment by Ryozuki** [comment]
close

**Comment by Learath2** [comment]
@ChillerDragon was this fixed?

**Comment by ChillerDragon** [comment]
@Learath2 no :c 
just tested with latest master and still not fixed. Oh btw it is easy to reproduce just join any 0.6.5 server and connect dummy.

**Comment by ChillerDragon** [comment]
This seems to be back. I just tested with a ddnet client at commit a96bf2b41ee3a4a5662627fa1df3593a6294d02a.
On a 0.6.5 pure teeworlds vanilla server. Connecting a dummy is not smooth at all.

The server I tested with is ddnet://45.142.178.158:8304

Okay but now I tested a bunch of times and it never got that bad again that I was fully stuck. So I guess this is good enough. I assume its annoying to debug. We have bigger problems right now than this.
